gravitee-apim-console-webui/src/management/application/details/subscribe/application-key.dialog.html

```diff
@@ -0,0 +1,38 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<md-dialog class="application-key-dialog">
+  <md-dialog-title class="application-key-dialog__title">
+    <h4>Select Key Type:</h4>
+  </md-dialog-title>
+  <md-dialog-content class="application-key-dialog__content">
+    <div class="application-key-dialog__content__details">
+      <p>API key is unique to each subscription.</p>
+      <p>Application key is shared across subscriptions.</p>
+    </div>
+    <div class="application-key-dialog__content__warning">
+      <p>Please note that this action is irreversible.</p>
+    </div>
+  </md-dialog-content>
+  <md-dialog-actions>
+    <div class="application-key-dialog__actions">
+      <md-button class="md-primary" ng-click="$ctrl.hide()">RETURN</md-button>
+      <md-button class="md-primary md-raised" ng-click="$ctrl.hide($ctrl.keyModes.EXCLUSIVE)">API KEY</md-button>
+      <md-button class="md-primary md-raised" ng-click="$ctrl.hide($ctrl.keyModes.SHARED)">APPLICATION KEY</md-button>
```

Comment:
I'm not sure of the wording we should use here, to name both modes.
It's not the same wording as the portal mockups.
Imo, "Application key" is confusing, as it's still an API key.

Should we use :
- Exclusive API Key
- Shared API Key

@mouligno : Any opinion ?

Reply:
Shared API Key / API-key (since the shared mode is to enable, let's not introduce useless complexity) don't you think ?

Reply:
Yes, it sounds good to me :+1: